packages/css-engine/src/core/to-value.ts

```diff
@@ -74,6 +74,10 @@ export const toValue = (
     return value.value.map((v) => toValue(v, options)).join(",");
   }
 
+  if (value.type === "tuple") {
+    return `${toValue(value.value[0])} ${toValue(value.value[1])}`;
```

Comment:
value.value is array, why not `map(toValue).join(' ')`

Reply:
done